src/index.html

```diff
@@ -5,8 +5,362 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="top-bar">
+      <a href="#" class="top-bar__logo">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="img_logo"
+        >
+      </a>
+
+      <nav class="top-bar__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link "
+               data-qa="nav-hover">laptops & computers</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="catalog">
+        <div
+           class="card"
+           data-qa="card"
+        >
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina"
+          >
+
+          <div class="card__title title" >
+            <h2 class="title__text" >APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+            <p class="title__code" >Product code: 195434</p>
+          </div>
+
+          <div class="review card__review">
+            <div class="stars stars--4">
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
+
+            <p class="review__count">Reviews: 5</p>
+          </div>
+
+          <div class="price card__price">
+            <p class="price__text">Price:</p>
+
+            <p class="price__amount">$2,199</p>
+          </div>
+
+          <a
+            class="card__button"
+            data-qa="card-hover"
+            href="#"
+          >
+             buy
+          </a>
```

Comment:
Pay attention on indents - fix all similar places
```suggestion
          >
            buy
          </a>
```

Reply:
https://github.com/mate-academy/layout_catalog/pull/3437/files#r1277476801
the same